akka-remote/src/test/scala/akka/remote/artery/TwoNodeArterySpec.scala

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright (C) 2009-2016 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka.remote.artery
+
+import akka.actor.{ ActorSystem, ExtendedActorSystem }
+import akka.testkit.AkkaSpec
+import com.typesafe.config.{ Config, ConfigFactory }
+
+object TwoNodeArterySpec {
+  def defaultConfig =
+    ConfigFactory.parseString("""
+      akka {
+        actor.provider = "akka.remote.RemoteActorRefProvider"
+        actor.warn-about-java-serializer-usage = off
+        remote.artery {
+          enabled = on
+          hostname = localhost
+          port = 0
+        }
+      }
+    """)
+}
+
+/**
+ * Base class for remoting tests what needs to test interaction between two actor systems
+ * communicating over artery.
+ */
+abstract class TwoNodeArterySpec(config: Config) extends AkkaSpec(config) {
```

Comment:
Yes, that is many times useful. Although it might be better to instead of creating a single system here and many accessor methods, expose a factory that does all the plumbing and returns the created system and its address. Rationale is that there are tests that use more than one system and it will become confusing if two is magically created in the background while the rest is created manually.